bench-tps/src/bench_tps_client/bank_client.rs

```diff
@@ -82,4 +83,16 @@ impl BenchTpsClient for BankClient {
         // BankClient doesn't support airdrops
         Err(BenchTpsError::AirdropFailure)
     }
+
+    fn get_account(&self, pubkey: &Pubkey) -> Result<Account> {
+        let account = SyncClient::get_account(self, pubkey).map_err(|err| err.into());
+        if let Ok(account) = account {
+            if account.is_none() {
+                return Err(BenchTpsError::Custom("Account was not found".to_string()));
+            }
+            Ok(account.unwrap())
+        } else {
+            Err(account.err().unwrap())
+        }
+    }
```

Comment:
```suggestion
       SyncClient::get_account(self, pubkey)
           .map_err(|err| err.into())
           .and_then(|account| account.ok_or(BenchTpsError::Custom("Account was not found".to_string())))
```
Simpler and more rust-y. (Suggestion probably needs formatting)

Reply:
Also, could we emulate the logic in RpcClient::get_account so we don't have to have this one-off custom error?